doc_source/autoscaling.md

```diff
@@ -189,6 +189,7 @@ Complete the following steps to deploy the Cluster Autoscaler\. We recommend tha
            - --cloud-provider=aws
            - --skip-nodes-with-local-storage=false
            - --expander=least-waste
+           # You will need to replace the <YOUR CLUSTER NAME> at the end of line.
            - --node-group-auto-discovery=asg:tag=k8s.io/cluster-autoscaler/enabled,k8s.io/cluster-autoscaler/<YOUR CLUSTER NAME>
```

Comment:
Need to scroll to the right to discover `<YOUR CLUSTER NAME>` at the end of line if user's screen does not wide enough when viewing the page https://docs.aws.amazon.com/eks/latest/userguide/autoscaling.html#cluster-autoscaler .